python/raf/frontend/pytorch.py

```diff
@@ -15,19 +15,16 @@
 from ..frontend.model import FrameworkModel
 
 
-def trace_model(model, input_type, input_shape):
+def trace_model(model, shape_dict):
     """Trace PyTorch model.
 
     Parameters
     ----------
     model: torch.nn.Module
         The PyTorch module to be converted.
 
-    input_type: str
-        Input type.
-
-    input_shape: Tuple[int, ...]
-        Input shape
+    shape_dict: Dict[str, Tuple[Tuple[int, ...], str, int(optional)]]
```

Comment:
This is not a valid type annotation.
```python
Dict[str, Union[Tuple[Tuple[int, ...]], int]]
```
Note that there's no way (and no need) to specify optional entry of a dict parameter in Python.